src/main/java/io/cdap/e2e/pages/locators/CdfLogLocators.java

```diff
@@ -60,4 +60,7 @@ public static WebElement locateLogEntry(String logLevel, String logMessage) {
       "//following-sibling::div[@data-cy='log-message'][contains(normalize-space(.), \"" + logMessage + "\")]";
     return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
   }
+
+  @FindBy(xpath = "//div[@data-cy='log-viewer-row']//div[contains(text(),'ERROR')]")
+  public static WebElement logError;
```

Comment:
can we make it more generic as a method named `locateLogLevel` which takes `logLevel` as an input?

Reply:
Change done.